mgmt_cli_test.py

```diff
@@ -1233,3 +1232,135 @@ def test_manager_installed_and_functional(self):
             assert host_health.rest_status == HostRestStatus.UP, "Host REST status is not 'UP'"
 
         self.log.info('finishing test_manager_installed_and_functional')
+
+    # ----- RESTORE -----
+
+    @staticmethod
+    def get_snapshot_data(snapshot_name: str):
+        snapshots_config = "defaults/manager_restore_benchmark_snapshots.yaml"
+        with open(snapshots_config, encoding="utf-8") as snapshots_yaml:
+            all_snapshots_dict = yaml.safe_load(snapshots_yaml)
+
+        try:
+            snapshot_dict = all_snapshots_dict["sizes"][snapshot_name]
+        except KeyError:
+            raise ValueError(f"Snapshot data for size '{snapshot_name}'GB was not found in the {snapshots_config} file")
+
+        snapshot_data = SnapshotData(
+            bucket=all_snapshots_dict["bucket"],
+            tag=snapshot_dict["tag"],
+            exp_timeout=snapshot_dict["exp_timeout"],
+            keyspaces=list(snapshot_dict["schema"].keys()),
+            cs_read_cmd_template=all_snapshots_dict["cs_read_cmd_template"],
+            prohibit_verification_read=snapshot_dict["prohibit_verification_read"],
+            number_of_rows=snapshot_dict["number_of_rows"],
+        )
+        return snapshot_data
+
+    def prepare_stress_read_cmd(self, command_template, keyspace_name, number_of_rows):
```

Comment:
`prepare_stress...` while it actually runs the stress, let's be precise on method actions.

Reply:
renamed to `prepare_and_run_stress_read`